python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -747,6 +757,8 @@ def run_transformer_int4_gpu_win(repo_id,
     from omegaconf import OmegaConf
     conf = OmegaConf.load(f'{current_dir}/config.yaml')
     today = date.today()
+    if 'exclude' in conf:
+        excludes = conf['exclude']
```

Comment:
I think we can process exclude here?
Process exclude to be a list of tuples (model, input_len)?
And then should_exclude is not needed, you only need to check if the tuple is in the list?